examples/linearCylinderPlaneStrain/linearCylinderPlaneStrain.m

```diff
@@ -0,0 +1,149 @@
+%md# Linear cylinder plane strain  example  
+%md
+%md[![Octave script](https://img.shields.io/badge/script-url-blue)](https://github.com/ONSAS/ONSAS.m/blob/master/examples/linearPlaneStrain/onsasExample_linearPlaneStrain.m)
+%md
+%md In this example a hollow cylinder submitted to an internal pressure $p_i$ Pa as shown in the diagram depcited below is considered. The length of the cylinder is $L_z $ m and the internal and external radios are $R_i$ m and $R_e$ m, respectively. 
+%md
+%md```@raw html
+%md<img src="../../assets/linearCylinderPlaneStrain/ilusCylinderPlaneStrain.svg" alt="linear cylinder diagram" width="500"/>
+%md```
+%md
+%md A cylindrical system of coordinates is defined considering the unitary vectors ($e_r$, $e_\theta$, $e_z$). 
+%md The material employed is isotropic and homogeneous with elasticity modulus $E=1$ MPa and Poisson's coefficient $\nu=0.3$. The pressure is radial and applied on the internal surface $\mathbf{\mathit{p_i}} =  - p_i \, e_r$. The boundary conditions correspond to this plane strain example holds:
+%md```math 
+%md \mathbf{\mathit{u}}_z(r, \theta, z=0)=0\;\;\quad\forall\; (r,\theta) \\
+%md u_z(r, \theta, z=L_z)=0\;\;\quad\forall\ (r,\theta)
+%md```
+%md
+%md## Analytic solution
+%md
+%md The analytical solution is obtained using the Navier's equation, imposing no temperature variation and no volumetric forces field. Thus the displacements field $\mathbf{\mathit{u}}$ solution satisfies:
+%md```math
+%md  \nabla (\nabla . u(r,\theta,z) )  = 0  
```

Comment:
I am not sure if Navier's equation would be this... and it would also be complicate to write also how the boundary conditions relate to the unknown $u$... I would suggest simply cite Timoshenko and thats it...

Reply:
![Screenshot-20220519220012-738x119](https://user-images.githubusercontent.com/50339940/169427876-bd8ab3f4-65c3-4da8-a6af-9e8ded242fc4.png)

Reply:
there u is a vector displacement field... then... you could use bold in u so that is not confused with the other u's you are using...

Reply:
and as I said you need to introduce also the boundary conditions explanation in order to show how the material parameters come into play in the solution (since navier's equations does not have any mechanical parameter!)

Reply:
Ok, you want me to include this:

CB:
```math
\mathbf{\mathit{u}}_z(r, \theta, z=0)=0\;\;\quad\forall\; (r,\theta) \\
```

```math
\textbf{\mathbf{T}} = -\lambda tr(\textbf{\mathbf{D}}) \textbf{\mathbf{I}} +  2 \mu \textbf{\mathbf{D}}
```

Reply:
Done